tests/spec/spec/kubernetes_master/kubernetes_master_spec.rb

```diff
@@ -207,18 +207,20 @@
   end
 end
 
-describe 'Check the docker cgroup and logging driver' do
-  describe file('/etc/docker/daemon.json') do
+describe 'Check the containerd' do
+  describe command('crictl --runtime-endpoint unix:///run/containerd/containerd.sock version') do
```

Comment:
Do we need `--runtime-endpoint unix:///run/containerd/containerd.sock`?

On [GitHub](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md#usage) they say:

> crictl by default connects on Unix to:
> - unix:///var/run/dockershim.sock or
> - unix:///run/containerd/containerd.sock or
> - unix:///run/crio/crio.sock or
> - unix:///var/run/cri-dockerd.sock

Reply:
It's better to run it that way due to the error msg (unix:///var/run/dockershim.sock is not found) since it's the default one.

Reply:
Does it mean that if one will run any `crictl` command (without `--runtime-endpoint` option) they get error message? If yes, I would opt for adding `/etc/crictl.yaml` file with `runtime-endpoint` setting. This can be done in separate task.

Reply:
Yes, it's not critical but I would add this config file as well.
 ```sh
[root@rafzei-offlinerhel-kubernetes-master-vm-0 ~]# crictl ps
WARN[0000] runtime connect using default endpoints: [unix:///var/run/dockershim.sock unix:///run/containerd/containerd.sock unix:///run/crio/crio.sock]. As the default settings are now deprecated, you should set the endpoint instead. 
ERRO[0002] connect endpoint 'unix:///var/run/dockershim.sock', make sure you are running as root and the endpoint has been started: context deadline exceeded 
WARN[0002] image connect using default endpoints: [unix:///var/run/dockershim.sock unix:///run/containerd/containerd.sock unix:///run/crio/crio.sock]. As the default settings are now deprecated, you should set the endpoint instead. 
ERRO[0004] connect endpoint 'unix:///var/run/dockershim.sock', make sure you are running as root and the endpoint has been started: context deadline exceeded 
CONTAINER           IMAGE               CREATED             STATE               NAME                        ATTEMPT             POD ID
51607730efffa       7801cfc6d5c07       22 hours ago        Running             dashboard-metrics-scraper   0                   8e729d8bfde2b
172b40d807752       e1482a24335a6       22 hours ago        Running             kubernetes-dashboard        0                   ea3a94870b3bc
3551a3e71749b       8d147537fb7d1       22 hours ago        Running             coredns                     0                   f4c91aaa261ec
6a1b586e28e66       8d147537fb7d1       22 hours ago        Running             coredns                     0                   fd2052a586fbf
286475b417f46       8522d622299ca       22 hours ago        Running             kube-flannel                0                   5d93eefaf8fdb
eb39650149a7f       edeff87e48029       22 hours ago        Running             kube-proxy                  0                   f13ff75abafd4
9b2406f358f9a       8a5cc299272d9       22 hours ago        Running             kube-apiserver              0                   be63ce94d6330
551798081a01a       721ba97f54a65       22 hours ago        Running             kube-scheduler              0                   45f151bf9d334
f81f3b784e732       0ce02f92d3e43       22 hours ago        Running             kube-controller-manager     0                   983e666a63d29
53fd6bddf24db       0048118155842       22 hours ago        Running             etcd                        0                   ad80b5df69cb2
```